geopandas/tests/test_explore.py

```diff
@@ -325,11 +325,11 @@ def test_no_crs(self):
     def test_style_kwds(self):
         """Style keywords"""
         m = self.world.explore(
-            style_kwds=dict(fillOpacity=0.1, weight=0.5, fillColor="orange")
+            style_kwds={"fillOpacity": 0.1, "weight": 0.5, "fillColor": "orange"}
```

Comment:
Sidenote: personally, for passing extra arguments, I find the dict form nicer because it resembles more normal arguments. 

(now, I am fine here for our tests to just follow ruff, I just wouldn't like that for my personal scripts ;))

Reply:
I also saw this and would agree preparing kwargs is more readable using `dict`. The other case like this I thought was not ideally is some `all()` assertions have lists converted to generators - but if you needed to inspect the test, the list form would be more useful. But am also happy to just use the ruff convention.

Reply:
Happy to proceed as is and get this merged, nothing stopping us reviewing this down the track